services/web/server/src/simcore_service_webserver/projects/projects_db.py

```diff
@@ -212,9 +216,62 @@ def engine(self) -> Engine:
             async for row in conn.execute(query):
                 result_dict = dict(row.items())
                 log.debug("found project: %s", result_dict)
+                tags = await self._get_tags_by_study(study_id=result_dict['id'])
+                result_dict['tags'] = tags
                 projects_list.append(_convert_to_schema_names(result_dict))
         return projects_list
 
+    async def _get_study(self, user_id: str, project_uuid: str, exclude_foreign: List = []) -> Dict: # pylint: disable=dangerous-default-value
```

Comment:
Pylint is warning you and you are looking another way  :-) ! Do not use mutables as defaults. Read  [this](https://docs.python-guide.org/writing/gotchas/#mutable-default-arguments) !

A workaround
```python
async def _get_study( ... , exclude_foreign: Optional[List]=None):
  exclude_foreign = exclude_foreign or []
  ...
```

Reply:
I didn't look another way! I read the thing and I politely told him to be quiet as the issue is not affecting this particular use (the variable is never written). Isn't modifying input parameters a bad practice in general anyway?